### PR TITLE
Add ability for user to change and open the GitHub repo of the project

### PR DESCRIPTION
**Pull request by johnmcmonigle**
It will also be saved to the disk

### CODE REVIEW

**Comment by johnmcmonigle** [comment]
Working on fixing merge conflicts

File: src/main/java/cpp/storage/Storage.java [hunk 1]
```diff
@@ -49,28 +49,30 @@ private static ProjectList readData() throws IOException {
...
             //add project to list
             projectList.addProject(title);
 
             String[] todoInfo = todos.split(",");
             //add todo to project
-            for (int i = 0; i < (todoInfo.length) / 2; i++) {
-                String todoDescrip = todoInfo[2 * i];
-                String todoStatus = todoInfo[2 * i + 1];
-                String todoDeadline = todoInfo[2 * i + 2];
+            for (int i = 0; i < (todoInfo.length - 1) / 2; i++) {
```
Comment (YaxinJoy) [inline src/main/java/cpp/storage/Storage.java:72]: It should not be `(todoInfo.length-1)/2`. If there are three elements saved in each todo, it should be `(todoInfo.length)/3`.

Comment (johnmcmonigle) [reply src/main/java/cpp/storage/Storage.java]: Fixed 👍

File: text-ui-test/EXPECTED.TXT [hunk 2]
```diff
@@ -16,13 +16,20 @@ ____________________________________________________________
...
+7. addtododeadline [Project Index] [Todo Index] [Date]
+8. todo [Project Index] [Description]
+9. mark [Project Index] [Todo Index]
+10. listprojects/listproject
+11. view [Project Name]
+12. exit
+____________________________________________________________
+What can I help you with next?
+____________________________________________________________
+
```
Comment (FTang21) [inline text-ui-test/EXPECTED.TXT:32]: Error might be coming from this? Not sure if "What can I help you with next?" is duplicated here

**Pull request merged**